app/models/container_node.rb

```diff
@@ -75,6 +75,12 @@ def ipaddress
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    if MiqRegion.my_region.role_active?("cockpit_ws")
+      uri = URI("/dashboard/cockpit_redirect")
+      uri.query = {:ip => ipaddress }.to_query
+    else
+      uri = URI::HTTP.build(:host => ipaddress, :port => 9090)
```

Comment:
The whole point of this PR is to move away from these. These are what is currently in master